docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -152,6 +152,12 @@ PersistentVolume Claims. Note that, the PersistentVolumes associated with the
 Pods' PersistentVolume Claims are not deleted when the Pods, or StatefulSet are deleted.
 This must be done manually.
 
+### Pod Name Label
+
+When the StatefulSet controller creates a Pod, it adds a label, `statefulset.kubernetes.io/pod-name`, 
+that is set to the name of the Pod. This label allows you to attach an individual Service to 
```

Comment:
by "individual", do you mean "distinct" or "different"?

Reply:
@kow3ns Bumping for action on docs review

Reply:
This should be "different"?